controllers/v1/criteria.js

```diff
@@ -0,0 +1,1319 @@
+const config = require('../../config/config');
+const rp = require('request-promise');
+const request = require('request');
+const model = require('../../db');
+const helperFunc = require('../../helper/chart_data');
+const pdfHandler = require('../../helper/common_handler');
+const omit = require('object.omit');
+const url = require("url");
+const rimraf = require("rimraf");
+const fs = require('fs');
+const path = require('path');
+
+
+/**
+   * @api {post} /dhiti/api/v1/criteria/instanceReport 
```

Comment:
@deepakr123 - Please make the URL as /dhiti/api/v1/observations/instanceReportByCriteria